lab_1/main.py

```diff
@@ -2,43 +2,45 @@
 Lab 1
 A concordance extraction
 """
-
-
-def tokenize(text: str) -> list:
-    """
-    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
```

Comment:
Please, leave the function description as it is - use the already written template.